src/components/ModelizerTextView.vue

```diff
@@ -1,24 +1,38 @@
 <template>
-  <div class="modelizer-text-view"
+  <div class="modelizer-text-view row"
     data-cy="modelizer-text-view"
   >
-    <monaco-editor :viewType="viewType"/>
+    <file-explorer
+      class="q-ma-md col-2 overflow-auto"
+      :nodes="nodes"
+    />
+    <q-separator vertical inset />
+    <monaco-editor :viewType="viewType" class="col-auto" />
   </div>
 </template>
 
 <script setup>
+import { getTree } from 'src/composables/FileExplorer';
 import MonacoEditor from 'components/editor/MonacoEditor.vue';
+import FileExplorer from 'components/FileExplorer.vue';
 
 defineProps({
   viewType: String,
 });
-</script>
 
-<style lang="scss" scoped>
-.modelizer-text-view {
-  display: flex;
-  flex: 1;
-  flex-direction: column;
-  height: 100vh;
-}
-</style>
+// TODO : Hard-coded data to display FileEplorer component
```

Comment:
the TODO should be "remove hard-coded data when ... "

Reply:
done